test/contract/generatepin/generatePin.js

```diff
@@ -0,0 +1,103 @@
+const expect = require('chai').expect;
+const config = require('app/config');
+const FormatUrl = require('app/utils/FormatUrl');
+const request = require('supertest');
+
+const VALIDATION_SERVICE_URL = config.services.validation.url;
+
+/* eslint no-console: 0 no-unused-vars: 0 */
+describe.only('Pin Creation API Tests', function() {
```

Comment:
Done, '.only' removed. This missed by the precommit hook because the test didn't start with 'test'. Also renamed the test for the precommit hook to function.